lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -501,58 +482,15 @@ def _quoted_partman_schema
   end
 
   def _fully_qualified_table_name_for_partman(table)
```

Comment:
I wouldn't worry about this now, but just to put this in the back of your head: this method is probably a candidate for moving to the Relation struct.